play/plugin/src/main/kotlin/com/github/triplet/gradle/play/tasks/internal/PlayApiService.kt

```diff
@@ -37,11 +41,17 @@ internal abstract class PlayApiService @Inject constructor(
     private val editIdFileAndFriends
         get() = listOf(editIdFile, editIdFile.marked("commit"), editIdFile.marked("skipped"))
 
+    var tasksHasFailed = false
```

Comment:
Can you call this `buildFailed`? If that conflicts, then maybe `buildHasFailingTasks`.